list-publications/src/main/java/no/unit/nva/publication/query/ListPublishedPublicationsHandler.java

```diff
@@ -62,4 +61,15 @@ protected PublishedPublicationsResponse processInput(Void input, RequestInfo req
     protected Integer getSuccessStatusCode(Void input, PublishedPublicationsResponse output) {
         return HttpStatus.SC_OK;
     }
+
+    private static DynamoDBPublicationService getPublicationService() {
+        Environment environment = new Environment();
+        AmazonDynamoDB amazonDynamoDB = AmazonDynamoDBClientBuilder.standard()
+                .withRegion(environment.readEnv(AWS_REGION))
+                .build();
+        return new DynamoDBPublicationService(
+                amazonDynamoDB,
```

Comment:
If the arguments were in one line, I think the return statement would be more readable